kanidmd_web_ui/src/views/security.rs

```diff
@@ -46,10 +51,12 @@ enum State {
     Init,
     Waiting,
     Error { emsg: String, kopid: Option<String> },
+    Success { msg: String },
 }
 
 pub struct SecurityApp {
     state: State,
+    unix_input_value: String,
```

Comment:
This feels like it should be an `Option<String>` just so we're never passing *any* string around unless it's actually in use.

Reply:
This became obsolete due to refactoring